frontend/apps/filemanager/filemanagermenu.lua

```diff
@@ -231,7 +249,7 @@ function FileManagerMenu:setUpdateItemTable()
                         end,
                     },
                     {
-                        text = _("Shrink item font size to fit more text"),
+                        text = _("Auto reduce filename-font-size to fit whole text"),
```

Comment:
Proposed change: "Auto reduce font size to fit more text" (we do not guarantee to fit the **whole** text).